src/qt/psbtoperationsdialog.cpp

```diff
@@ -17,6 +17,10 @@
 
 #include <iostream>
 
+using node::AnalyzePSBT;
+using node::BroadcastTransaction;
```

Comment:
In commit "Add src/node/* code to node:: namespace" (733a629696ef3350e5c9c861ae94254a06cc920b)

Note: This line can be removed after #23499, since calling `node::BroadcastTransaction` from the bitcoin-gui process doesn't work in the multiprocess build (there is a segfault because the NodeContext pointer this tries to pass is null). This is a preexisting bug, independent of this PR.